### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by danpersa**
Improve the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,50 @@ Skrop is a media service based on [Skipper](https://github.com/zalando/skipper)
...
+Here is how a route from the `sample.eskip` file would look like:
+ 
+```
+small: Path("/images/S/:image")
+  -> modPath("^/images/S", "/images")
+  -> longerEdgeResize(800)
+  -> "http://localhost:9090";
+```
+
+What this is means, is that when somebody does a `GET` to `http://skrop.url/images/S/myimage.jpg`,
```
Comment (mfellner) [inline README.md]: "What this means"

File: README.md [hunk 2]
```diff
@@ -4,15 +4,50 @@ Skrop is a media service based on [Skipper](https://github.com/zalando/skipper)
...
+Skrop will do a call to `http://localhost:9090/images/myimage.jpg`, retrieve
```
Comment (mfellner) [inline README.md]: "to retrieve"

File: README.md [hunk 3]
```diff
@@ -4,15 +4,50 @@ Skrop is a media service based on [Skipper](https://github.com/zalando/skipper)
...
+the resized image back in the response.
+
+
+## Filters
+
+Skrop provides a set of filters, which you can use within the routes:
+
+```
+longerEdgeResize(size) — resizes the image to have the longer edge as specified,
+                         while the same time preserving the aspect ratio
```
Comment (mfellner) [inline README.md]: "while at the same time"

Comment (danpersa) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (danpersa) [inline README.md]: done

File: README.md (hunk 1 above)
Comment (danpersa) [inline README.md]: done

**Pull request merged**